src/Cli/dotnet/commands/dotnet-test/TestingPlatformCommand.cs

```diff
@@ -102,6 +116,38 @@ public int Run(ParseResult parseResult)
             return 0;
         }
 
+        private static IEnumerable<string> GetMatchedModulePaths(string testModules, string rootDirectory)
+        {
+            var testModulePatterns = testModules.Split(new[] { ';' }, StringSplitOptions.RemoveEmptyEntries);
+
+            Matcher matcher = new();
+            matcher.AddIncludePatterns(testModulePatterns);
+
+            PatternMatchingResult result = matcher.Execute(
+                new DirectoryInfoWrapper(
+                    new DirectoryInfo(rootDirectory)));
+
+            return result.Files.Select(file => $"{rootDirectory}\\{file.Path.Replace("/", "\\")}");
```

Comment:
Is it fine for Linux to normalize to backslash? It's usually easier to normalize on the most restrictive (i.e. forward slash).